src/components/Notification.tsx

```diff
@@ -0,0 +1,80 @@
+import classnames from 'classnames';
+import { HeadingLevel } from '../types/headingLevel';
+import { Icon } from './Icon';
+import './Notification.css';
+import {
+  NotificationLink,
+  NotificationLinkProperties
+} from './NotificationLink';
+
+interface NotificationProperties {
+  type: 'success' | 'warning' | 'error' | 'info' | 'loading';
+  message?: React.ReactNode;
+  headingLevel?: HeadingLevel;
+  children?: React.ReactNode;
+  links?: Array<NotificationLinkProperties>;
+}
+
+/**
+ * CFPB DS Notification
+ *
+ * https://cfpb.github.io/design-system/components/notifications
+ *
+ * @param children Notification explaination
+ * @param className Any additional classnames for the notification wrapper
+ * @param headingLevel Controls sizing of primary message
+ * @param links Links
+ * @param message Notification reason
+ * @param type Type of notification
+ * @returns ReactElement
+ */
+export const Notification = ({
+  children,
+  className,
+  headingLevel = 'h4',
+  links,
+  message,
+  type = 'info',
+  ...props
+}: NotificationProperties & React.HTMLAttributes<HTMLDivElement>) => {
```

Comment:
Missing a return type here:

```suggestion
}: NotificationProperties & React.HTMLAttributes<HTMLDivElement>): React.ReactElement => {
```